src/main/java/entities/Ingredient.java

```diff
@@ -0,0 +1,4 @@
+package entities;
+
+public class Ingredient {
```

Comment:
The CRC card says, that this class "stores the name of the Ingredient". We should add that instance variable as well as getters and setters.